parking-management/src/main/kotlin/com/touk/parkingmanagement/parking/ParkingController.kt

```diff
@@ -0,0 +1,41 @@
+package com.touk.parkingmanagement.parking
+
+import com.touk.parkingmanagement.parking.protocol.ParkingFeeRequest
+import com.touk.parkingmanagement.parking.protocol.ParkingFeeResponse
+import com.touk.parkingmanagement.parking.protocol.StartParkingRequest
+import com.touk.parkingmanagement.parking.protocol.StopParkingRequest
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.net.URI
+import javax.validation.Valid
+
+@RestController
+@RequestMapping("/api/parkings")
+class ParkingController(
+    private val parkingFacade: ParkingFacade
+) {
+
+    @PostMapping
+    fun startParking(@RequestBody @Valid startParkingRequest: StartParkingRequest): ResponseEntity<Any> {
+        val rentalId = parkingFacade.startParking(startParkingRequest.vehicleId, startParkingRequest.driverId)
+        return ResponseEntity.created(URI.create("/api/parkings/$rentalId")).build()
+    }
+
+    @PutMapping("/{parkingId}/stop")
```

Comment:
czy `stop` to dobra część ścieżki w REST?

Reply:
Nie jest. Powinien być odpowiedni rzeczownik i w ciele komenda do wywołania zatrzymania.